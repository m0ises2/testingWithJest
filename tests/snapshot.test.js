const rick = require('../rick');
const getCharacter = require('../snapshot');

describe('Testing snapshots', () => {
    test('Es igual', () => {
        expect(getCharacter(rick)).toMatchSnapshot();
    });
});
