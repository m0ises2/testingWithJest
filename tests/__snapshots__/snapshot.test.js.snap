// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing snapshots Es igual 1`] = `
Object {
  "gender": "Males",
  "id": 1,
  "name": "Rick Sanchez",
  "species": "Human",
  "status": "Alive",
}
`;
